citation/paper.md

```diff
@@ -0,0 +1,146 @@
+---
+title: matchms - processing and similarity evaluation of mass spectrometry data.
+tags:
+  - Python
+  - mass spectrometry
+  - metadata cleaning
+  - data processing
+  - similarity measures
+
+authors:
+  - name: Florian Huber
+    orcid: 0000-0002-3535-9406
+    affiliation: 1
+  - name: Stefan Verhoeven
+    orcid: 0000-0002-5821-2060
+    affiliation: 1
+  - name: Christiaan Meijer
+    orcid: 0000-0002-5529-5761
+    affiliation: 1
+  - name: Hanno Spreeuw
+    orcid: 0000-0002-5057-0322
+    affiliation: 1
+  - name: Cunliang Geng
+    orcid: 0000-0002-1409-8358
+    affiliation: 1
+  - name: Simon Rogers
+    orcid: 0000-0003-3578-4477
+    affiliation: 2
+  - name: Justin J. J. van der Hooft
+    orcid: 0000-0002-9340-5511
+    affiliation: 3
+  - name: Adam Belloum
+    orcid: 0000-0001-6306-6937
+    affiliation: 1
+  - name: Faruk Diblen
+    orcid: 0000-0002-0989-929X
+    affiliation: 1
+  - name: Juriaan H. Spaaks
+    orcid: 0000-0002-7064-4069
+    affiliation: 1
+
+affiliations:
+ - name: Netherlands eScience Center, Science Park 140, 1098XG Amsterdam, The Netherlands
+   index: 1
+ - name: School of Computing Science, University of Glasgow, Glasgow, United Kingdom
+   index: 2
+ - name: Bioinformatics Group, Plant Sciences Group, University of Wageningen, Wageningen, the Netherlands
+   index: 3
+date: 16 June 2020
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Mass spectrometry data is at the heart of numerable applications in the biomedical and life sciences.
+With growing use of high throughput techniques, but also through joint effort in the research community, fragmentation mass spectrometry datasets are growing in size and number.
+Platforms such as GNPS serve as an open-access hub for sharing of raw, processed, or annotated fragmentation mass spectrometry data (MS/MS) [@Wang2016].
```

Comment:
[MetaboLights](https://www.ebi.ac.uk/metabolights) is the European hub for metabolomics. We could also mention it besides to GNPS.

Reply:
True, thanks! I added it as well.

Reply:
great